grails-forge-core/src/main/java/org/grails/forge/feature/build/gitignore.rocker.raw

```diff
@@ -1,14 +1,37 @@
-Thumbs.db
-.DS_Store
+HELP.md
 .gradle
 build/
-target/
```

Comment:
I think we should keep the following vs removing them, so that this change does unintentionally cause files to be included which were ignored historically.  Something like:

```
### Other ###
Thumbs.db
.DS_Store
target/
```